includes/integration.php

```diff
@@ -799,20 +804,25 @@ public function new_order_event($order_id){
         $order = new WC_Order($order_id);
 
         $call_params = false;
-
-        if(!$this->stringIsPresent(get_post_meta($order_id, '_billing_email', true))) {
+    
+        if(!$this->stringIsPresent(get_post_meta($order_id, '_billing_email', true)) && !$this->stringIsPresent(get_post_meta($order_id, '_billing_phone', true))) {
             return;
         }
-
+    
+        $email     = get_post_meta($order_id, '_billing_email', true);
+        $phone     = get_post_meta($order_id, '_billing_phone', true);
+        $firstName = get_post_meta($order_id, '_billing_first_name', true);
+        $lastName  = get_post_meta($order_id, '_billing_last_name', true);
+    
         // identify user - put identify data in cookie
         $this->identify_call_data = array(
-            'id'        => get_post_meta($order_id, '_billing_email', true),
+            'id'        => $email ? $email : $phone . '@phone_email',
             'params'    => array(
-                        'email' 		=> get_post_meta($order_id, '_billing_email', true),
-                        'first_name' 	=> get_post_meta($order_id, '_billing_first_name', true),
-                        'last_name' 	=> get_post_meta($order_id, '_billing_last_name', true),
-                        'name'			=> get_post_meta($order_id, '_billing_first_name', true) . ' ' . get_post_meta($order_id, '_billing_last_name', true),
-                    )
+                'email' 		=> $email ? $email : $phone . '@phone_email',
```

Comment:
```php
'id'        => $email ? $email : $phone . '@phone_email',
...
'email' 		=> $email ? $email : $phone . '@phone_email',
```
those look pretty similar to each other.

Reply:
Refactored.